Makefile

```
@@ -86,7 +86,10 @@ qenerate: gql-introspection gql-query-classes
 localstack:
 	@$(CONTAINER_ENGINE) compose -f dev/localstack/docker-compose.yml up
 
-all-tests: linter-test types-test qenerate-test helm-test unittest
+all-tests: uv-lock-test linter-test types-test qenerate-test helm-test unittest
```

checkmake (0.3.2)

[warning] Target "all-tests" should be declared PHONY.

(phonydeclared)